back/engines/commercial/user_custom_fields/app/models/user_custom_fields/patches/custom_field.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module UserCustomFields::Patches::CustomField
+  def self.included(base)
+    base.class_eval do
+      # TODO: Refactor CustomFields to use STI with two types: UserCustomField & FormCustomField
```

Comment:
I would write "TODO tech debt:" at the beginning of this comment, because for the reader it is not clear whether it is a to-do for you, or a general to-do, i.e. tech debt.